Replace multicast address with class E (#5051)
istio/istio.io: content/en/docs/setup/install/multicluster/gateways/index.md

    {{< /tip >}}

    If the global services have actual VIPs, you can use those, but otherwise we suggest

>     using IPs from the class E addresses range `240.0.0.0/4` that are preserved.

Reviewer: What does "are preserved" mean? How can one know which ones are preserved?

Author: `Class E networks are defined by having the first four network address bits as 1. That encompasses addresses from 240.0.0.0 to 255.255.255.255. While this class is reserved, its usage was never defined. As a result, most network implementations discard these addresses as illegal or undefined. The exception is 255.255.255.255, which is used as a broadcast address.`

>     These IPs are not loopback addresses and are non-routable outside of a pod.    

Reviewer: ```suggestion
```

>     These IPs are not loopback addresses and are non-routable outside of a pod.    

Reviewer: Maybe delete this sentence and add to the Note below?

Author: ok

    Application traffic for these IPs will be captured by the sidecar and routed to the
    appropriate remote service.
    

>     **Note** Multicast addresses(224.0.0.0 ~ 239.255.255.255) should not be used as there is no route to them by default.

Reviewer: ```suggestion
{{< warning >}}
Use IPs that are not loopback addresses but also not routable outside of a pod.
Do not use multicast addresses (224.0.0.0 ~ 239.255.255.255) because there is no route to them by default.
{{< /warning >}}
```

    {{< /tip >}}

    If the global services have actual VIPs, you can use those, but otherwise we suggest

>     using IPs from the class E addresses range `240.0.0.0/4`, because they are preserved.

Reviewer: I suggest leaving off the "because ..." part.
```suggestion
    using IPs from the class E addresses range `240.0.0.0/4`.
```

    Application traffic for these IPs will be captured by the sidecar and routed to the
    appropriate remote service.

    {{< warning >}}

>     Multicast addresses(224.0.0.0 ~ 239.255.255.255) should not be used as there is no route to them by default.

Reviewer: ```suggestion
    Multicast addresses (224.0.0.0 ~ 239.255.255.255) should not be used because there is no route to them by default.
```

>     Loopback addresses range `127.0.0.0/8` should also not be used, as the traffic sent to lo may be redirected to inbound listener.

Reviewer: ```suggestion
    Loopback addresses (127.0.0.0/8) should also not be used because traffic sent to them may be redirected to the sidecar inbound listener.
```
